Tests/SKSwiftPMWorkspaceTests/SwiftPMWorkspaceTests.swift

```diff
@@ -111,11 +112,11 @@ final class SwiftPMWorkspaceTests: XCTestCase {
 
     check("-target", arguments: arguments) // Only one!
 #if os(macOS)
-    check("-target", "x86_64-apple-macosx10.10", arguments: arguments)
+    check("-target", Triple.hostTriple.tripleString(forPlatformVersion: "10.10"), arguments: arguments)
```

Comment:
I wonder if due to this issue: https://twitter.com/jckarter/status/1095757933579689985 we shouldn't stick to minimum 10.11 on macOS?

Reply:
If this ever breaks we can change it, but this test won't try to generate code, so I think it's probably fine.